controllers/maincontroller.js

```diff
@@ -1,5 +1,4 @@
 const IntroModel=require('../models/intromodel');
-const User = require('../models/user');
```

Comment:
We need this to give parameter in rendering intro.ejs

Reply:
Oh sorry, reverted this change.

Reply:
Cool no issues!